### PR TITLE
Added Airfoil_Noise_Prediction.ipynb file

### PR DESCRIPTION
**Pull request by kampaitees**
Here first I had converted the data to readable format then check whether there are null values than did feature scaling then split the data into 80-20 ratio and finally applied the linear regression model to calculate the mean and log mean squared error.

### CODE REVIEW

**Comment by kampaitees** [comment]
Is it correct ?

**Comment by nihal-rao** [comment]
Hello @kampaitees , nice work with the notebook !
However, it is good practice to split the data into training and test set and then use `fit_transform` on the training data and only `transform` on the test data. This prevents what is known as **train-test contamination.**
An explanation can be found [here](https://www.kaggle.com/alexisbcook/data-leakage/data)
**Happy Coding!**

**Comment by kampaitees** [comment]
Oh! I split after applying fit_tranform

**Comment by kampaitees** [comment]
Can you plz merge it?

**Comment by SaurabhAgarwala** [comment]
@nihal-rao can this be merged?

**Comment by nihal-rao** [comment]
Yes, please do merge @SaurabhAgarwala
